src/app/shared/components/course-bar-item/course-bar-item.component.html

```diff
@@ -0,0 +1,24 @@
+<svg viewBox="0 0 100 100" preserveAspectRatio="none">
+  <polygon points="0, 0
```

Comment:
Should this be in the shared mode? In the newer design, I think this should fit into the course module.

Reply:
It made sense to me to put this in the shared mode because then the svg will scale itself to the div box that it is placed in. What do you think?